hack/cherry_pick_pull.sh

```diff
@@ -16,7 +16,8 @@
 
 # Checkout a PR from GitHub. (Yes, this is sitting in a Git tree. How
 # meta.) Assumes you care about pulls from remote "upstream" and
-# checks thems out to a branch named pull_12345.
+# checks thems out to a branch named:
+#  automated-cherry-pick-of-<pr>-<target branch>-<timestamp>
```

Comment:
The timestamp is supposed to make it unique -- according to the script.  If the branch somehow already exists, `git checkout -b` will fail and the script will exit.